vizro-core/docs/pages/user-guides/graph.md

```diff
@@ -67,7 +68,86 @@ To add a [`Graph`][vizro.models.Graph] to your page, do the following:
 
 In the Python example we directly inserted the pandas DataFrame `df` into `figure=px.scatter_matrix(df, ...)`. This is [one way to supply data to a chart](data.md#supply-directly). For the YAML version, we [refer to the data source by name](data.md#reference-by-name) as `data_frame: iris`. For a full explanation of the different methods you can use to send data to your dashboard, see [our guide to using data in Vizro](data.md).
 
+## Customize Plotly chart
+
+If you want to customize the Plotly chart beyond a function call—such as using post-update methods like
+`update_layout`, `update_xaxes`, or `update_traces`, or creating a custom `plotly.graph_objects.Figure()` object
+and manually adding traces with `add_trace`—you will need to create a custom chart.
```

Comment:
Too many `-` characters by my opinion 😄 What do you think about this:
```suggestion
If you want to customize the Plotly chart beyond a function call, such as using post-update methods like
`update_layout`, `update_xaxes`, `update_traces`, or creating a custom `plotly.graph_objects.Figure()` object
and manually adding traces with `add_trace`, you will need to create a custom chart.
```

Reply:
I've made a slight revision too, and it basically puts the action at the start of the sentence "you will need to create a custom chart..." then splits the reasons why you'd want to into bullets, so they're clear if someone is scanning the text because they have that particular intent. WDYT?

Reply:
I like it!